codegen/src/main/twirl/templates/ScalaClient/Client.scala.txt

```diff
@@ -91,3 +99,19 @@ object @{service.name}Client {
         .build()
   }
 }
+
+
+trait @{service.name}LiftedClient {
```

Comment:
While we've been speaking about 'lifted', I'm not sure I actually like calling it that. What do you think about `@{service.name}ClientPowerApi` or something similar?